.circleci/config.yml

```diff
@@ -17,3 +17,5 @@ jobs:
       - run: sbt +compile
       - run: sbt +test:compile
       - run: sbt +test
+      - run: sbt "dumpEventSchemaCompilerPlugin / publishLocal"
+      - run: sbt scripted
```

Comment:
😯 what's that?

Reply:
Automatic sbt plugin testing

Reply:
Wow and this step is called _`scripted`_? 😯 what's the logic behind this name? 🤔

Reply:
I'd actually specify separate `name`/`command` in this step to make clear what happens here, see e.g. https://github.com/VirtusLab/git-machete/blob/master/.circleci/config.yml#L13

Reply:
I have no idea what logic is behind 'scripted' name. You write a script that test a project, but why is a verb in past simple???